proof/refine/RISCV64/Finalise_R.thy

```diff
@@ -2206,11 +2196,6 @@ definition
   finaliseCapTrue_standin_simple_def:
   "finaliseCapTrue_standin cap fin \<equiv> finaliseCap cap fin True"
 
-context
-begin
-
-declare if_cong [cong]
```

Comment:
Thanks for removing that. This does not actually achieve anyway what it looks like it wants to do: even in a local context the `declare` remains global. 

Just for to reinforce the right pattern, to remove something locally, it'd be:

```
context
  notes if_cong[cong]
begin
...
end
```